examples/bukkit/kotlindsl/src/main/kotlin/io/github/jorelali/Main.kt

```diff
@@ -1,15 +1,10 @@
 package io.github.jorelali
 
-import dev.jorel.commandapi.CommandAPI
-import dev.jorel.commandapi.CommandAPIBukkitConfig
 import org.bukkit.plugin.java.JavaPlugin
 
 class Main : JavaPlugin() {
 
 	override fun onEnable() {
-		CommandAPI.onLoad(CommandAPIBukkitConfig(this))
-		CommandAPI.onEnable()
```

Comment:
Why did you remove these two lines? I feel like they shoud stay because the examples are supposed to present a possible way of creating a plugin using the CommandAPI.

Reply:
I removed them because they are only required when actually shading the plugin, but we are not. (https://commandapi.jorel.dev/8.8.0/shading.html).

Reply:
Thanks for pointing that out, I forgot what `commandapi-bukkit-core` is doing...